### DISCUSSION TITLE
Is there a way to "re-request review?"

### DISCUSSION THREAD

**Discussion by andres-lowrie** [pwntester/octo.nvim#390] [Q&A] [answered]
some reviews have a lot of back and forth between assignees and reviewers; in the web ui there's a button next to the reviewer that allows one to "re-request review" which will let the reviewer know they should do another pass

wondering if there's a way to do this, I couldn't find this in the helpdocs so apologies if I missed it

**Comment by pwntester** [comment, marked as answer]
Sorry for the late response. This is not implemented yet. There seems to be no API to do that though

**Reply by andres-lowrie** [reply]
no worries, we all got many a plate spinning!
